src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,9 +1,5 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.